_attributes/common-attributes.adoc

```diff
@@ -122,10 +122,10 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
 :SMProductShortName: Service Mesh
-:SMProductVersion: 2.1.2.1
+:SMProductVersion: 2.1.3
 :MaistraVersion: 2.1
 //Service Mesh v1
-:SMProductVersion1x: 1.1.18.1
+:SMProductVersion1x: 1.1.19
```

Comment:
@rh-tokeefe: The version for SM 1.1 is 1.1.18-2.

FYI: 1.1 Errata name: Red Hat OpenShift Service Mesh 1.1.18.2 Containers